CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+# [v1.3.0-alpha.0](https://github.com/kubermatic/kubeone/releases/tag/v1.3.0-alpha.0) - 2021-07-21
```

Comment:
Can you make sure that the changelog follows the usual format? We should have the following sections

* Attention Needed
* Added
* Changed
* Removed (if we removed anything)

Check the v1.2.0 changelog as an example: https://github.com/kubermatic/kubeone/blob/master/CHANGELOG.md#v120---2021-03-18

Also, please make sure to fix the URLs rendering.